profile/src/main/java/rdx/works/profile/domain/GetProfileUseCase.kt

```diff
@@ -80,6 +81,16 @@ suspend fun GetProfileUseCase.accountOnCurrentNetwork(
     account.address == withAddress
 }
 
+suspend fun GetProfileUseCase.nextDerivationPathForAccountOnNetwork(networkId: Int): DerivationPath {
```

Comment:
let's better keep it as it was before: `nextDerivationPathForAccountOnCurrentNetworkWithLedger` or even better `nextDerivationPathForLedgerAccountOnCurrentNetwork`

Reply:
the name used before is not correct. It is derivation path for networkId, not on current network. This part was main casue of the bug so I want naming to be precise

Reply:
oh right, actually I meant the part of the "ledger" `LedgerAccount` but the rest you are correct

Reply:
since we have global entity index per network, it is exactly same way to generate derivation path for software/hardware account, this is why I removed "ledger" from the name